src/host/test/ledger.cpp

```diff
@@ -1126,7 +1126,7 @@ TEST_CASE("Recovery resilience")
     ledger.commit(last_idx);
   }
 
-  SUBCASE("Corrupt table offset in committed chunk")
+  SUBCASE("Corrupt table offset in committed chunk has no effect")
```

Comment:
Important note: This unit test was testing something that is _no longer true_. We used to look at all ledger files, so would notice corruption in `.committed` files (and rename them `.ignored`, and end up restoring with a prefix of the uncorrupted source ledger). Now we don't look at `.committed` files, so corruption in them _has no effect_.